packages/google_sign_in/google_sign_in_web/lib/src/gis_client.dart

```diff
@@ -188,9 +188,9 @@ class GisSdkClient {
   //
   // Token clients have an additional `error_callback` for miscellaneous
   // errors, like "popup couldn't open" or "popup closed by user".
-  void _onTokenError(Object? error) {
+  void _onTokenError(GoogleIdentityServicesError? error) {
```

Comment:
Woops, this change also requires a changelog entry and pubspec version, sorry I didn't notice the separate package!

Reply:
No worries! I just wasn't sure if it makes sense to have a changelog here, but done!